src/adapters/http/server.ts

```diff
@@ -50,11 +50,11 @@ class HttpAdapter extends Adapter {
       })
       return {
         promise,
-        done: (val: boolean, code = 401, message = 'Unauthorized') => {
```

Comment:
can you do the same for the WebSocket adapter?

Reply:
@KhudaDad414 Sure :)